qc_other_datasets/generate_qc.sh

```diff
@@ -0,0 +1,297 @@
+#!/bin/bash
+#
+# Compare the CSA of soft GT thresholded at different values on spine-generic test dataset.
+# 
+# Adapted from: https://github.com/ivadomed/model_seg_sci/blob/main/baselines/comparison_with_other_methods_sc.sh
+#
+# Usage:
+#     sct_run_batch -config config.json
+#
+# Example of config.json:
+# {
+#  "path_data"   : "<PATH_TO_DATASET>",
+#  "path_output" : "<PATH_TO_DATASET>_2023-08-18",
+#  "script"      : "<PATH_TO_REPO>/model_seg_sci/baselines/comparison_with_other_methods.sh",
+#  "jobs"        : 8,
+#  "script_args" : "<PATH_TO_REPO>/model_seg_sci/packaging/run_inference_single_subject.py <PATH_TO_MODEL>/sci-multisite-model <PATH_TO_CONTRAST-AGNOSTIC_REPO>/monai/run_inference_single_image.py <PATH_TO_CONTRAST-AGNOSTIC_MODEL>"
```

Comment:
Very minor nitpick: you could update `model_seg_sci`

Reply:
done in https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/90/commits/894db9cf92ec43d11cc2b09a860badc5e4c6e802